src/MongoDB.Bson/Serialization/BsonClassMap.cs

```diff
@@ -426,6 +421,42 @@ public static void RegisterClassMap(BsonClassMap classMap)
             }
         }
 
+        // private static methods
+        private static Type GetFormatterServicesType()
+        {
+#if NET452
+            return typeof(FormatterServices);
```

Comment:
Don't actually need to use reflection on .NET Framework.

I think we might have been using reflection before just to have a single way to do it on both targets.